Common/Parsers/JSONParser.swift

```diff
@@ -469,155 +484,6 @@ extension ComplicationTimelineEntry: Encodable, Decodable {
- 
- let device = Device(rawValue: deviceString) ?? Device.Unknown
- let direction = Direction(rawValue: directionString) ?? Direction.None
- let noise = Noise(rawValue: noiseInt) ?? Noise()
- 
- let sensorValue = SensorGlucoseValue(direction: direction, device: device, rssi: rssi, unfiltered: unfiltered, filtered: filtered, mgdl: mgdl, noise: noise, milliseconds: mills)
- self.sgvs.insertOrUpdate(sensorValue)
- }
- }
- */
- 
- let mbgs = json[GlobalJSONKey.mbgs] as? [[String: Any]] ?? []
- // let mbgs = json[GlobalJSONKey.mbgs]
- 
- print("mbgs count: \(mbgs.count)")
- 
- for subJson in mbgs {
- if let deviceString = subJson[MeteredGlucoseValue.JSONKey.device] as? String,
- let mills = subJson[MeteredGlucoseValue.JSONKey.mills] as? Double,
- let mgdl = subJson[MeteredGlucoseValue.JSONKey.mgdl] as? Double {
- let device = Device(rawValue: deviceString) ?? .unknown
- let meter = MeteredGlucoseValue(milliseconds: mills, device: device, mgdl: mgdl)
- self.mbgs.insertOrUpdate(meter)
- }
- }
- 
- /*
- for (_, subJson) in mbgs {
- if let deviceString = subJson[MeteredGlucoseValue.JSONKey.device].string, let mills = subJson[MeteredGlucoseValue.JSONKey.mills].double, let mgdl = subJson[MeteredGlucoseValue.JSONKey.mgdl].double {
- let device = Device(rawValue: deviceString) ?? Device.Unknown
- let meter = MeteredGlucoseValue(milliseconds: mills, device: device, mgdl: mgdl)
- self.mbgs.insertOrUpdate(meter)
- }
- }
- */
- let cals = json[GlobalJSONKey.cals] as? [[String: Any]] ?? []
- // let cals = json[GlobalJSONKey.cals]
- print("cals count: \(cals.count)")
- 
- for subJson in cals {
- if let slope = subJson[Calibration.JSONKey.slope] as? Double,
- let intercept = subJson[Calibration.JSONKey.intercept] as? Double,
- let scale = subJson[Calibration.JSONKey.scale] as? Double,
- let mills = subJson[Calibration.JSONKey.mills] as? Double {
- let calibration = Calibration(slope: slope, intercept: intercept, scale: scale, milliseconds: mills)
- self.cals.insertOrUpdate(calibration)
- }
- }
- 
- /*
- for (_, subJson) in cals {
- if let slope = subJson[Calibration.JSONKey.slope].double, let intercept = subJson[Calibration.JSONKey.intercept].double, let scale = subJson[Calibration.JSONKey.scale].double, let mills = subJson[Calibration.JSONKey.mills].double {
- let calibration = Calibration(slope: slope, intercept: intercept, scale: scale, milliseconds: mills)
- self.cals.insertOrUpdate(calibration)
- }
- }
- */
- // makes sure things are sorted correctly by date. When delta's come in they might screw up the order.
- self.sgvs.sorted()
- self.cals.sorted()
- self.mbgs.sorted()
- self.deviceStatuses.sorted()
- 
- }
- }
- */
-
 
 
```

Comment:
sounds good. This was old parsing code for socket.io data. We are still using REST.